backend/coreapp/asm_diff_wrapper.py

```diff
@@ -43,6 +43,38 @@ def create_config(arch: asm_differ.ArchSettings) -> asm_differ.Config:
             algorithm="levenshtein",
         )
 
+    @staticmethod
+    def get_objdump_target_function_flags(sandbox: Sandbox, target_path, platform: str, label: str) -> List[str]:
+        if compiler_wrapper.supports_objdump_disassemble(platform):
```

Comment:
we should add `and label` to this check I think, since it might not always be non-empty, even if the platform supports it

Reply:
ah, `label` can be empty but not None, right... thoughts on moving the `if label` check into this function?

Reply:
Yeah, that's what I'm advocating for

Reply:
ohhh I see, it was outside of this function. my b. yeah just putting it here is good